release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,10 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 12 February 2024 - Fix - CloudGateway 2.13.5 - Stability degradation since CloudGateway 2.13.4 [ID_38730]
+
+Since CloudGateway 2.13.4, the CloudGateway service could silently crash but would be able to recover automatically. This issue has been resolved.
```

Comment:
We usually avoid the term "crash" (see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/crash).
```suggestion
Since CloudGateway 2.13.4, a problem could occur in the CloudGateway service that temporarily made it stop responding. The service was able to recover from this problem automatically. This issue has been resolved.
```